src/data/markdown/docs/01 guides/02 Using k6/14 Executors.md

```diff
@@ -0,0 +1,535 @@
+---
+title: Executors
```

Comment:
I think we should either call this guide "Scenarios" and rework the page a little bit to first talk about scenarios as a concept and explaining multi-scenario tests before going into the details of each executor which is a setting for a scenario, or we should introduce a new guide called "Scenarios" that is introduced before this guide that talks about scenarios as a concept etc. linking to this guide for more details.

Reply:
Makes sense. The feature as a whole is "scenarios", and "executors" are a (crucial) part that make them possible, so we should mention and list them, but not make it the focal point of the article.

I'll see what I can add in the intro, and we can expand it to a standalone article if needed.

Reply:
@robingustafsson Let me know if 17b520a is sufficient for what you had in mind. I think this is a gentler introduction, focusing on the benefits of scenarios and a general configuration example before going into details with executors. I'm not sure what else we can mention there, so maybe it doesn't need a separate article?